sdk/core/core-http/README.md

```diff
@@ -6,12 +6,16 @@ This is the core HTTP pipeline for Azure SDK JavaScript libraries which work in
 
 ### Requirements
 
-- Node.js version > 6.x
-- npm install -g typescript
+- [Node.js](https://nodejs.org) version > 8.x
+- Typescript compiler
+
+```shell
+npm install -g typescript
+```
 
 ### Installation
 
-- After cloning the repo, execute `npm install`
+This package is not meant to be consumed directly by end users.
```

Comment:
Perhaps a follow up a line regarding what it is meant for?

Reply:
It is already mentioned in the first paragraph so I felt it would be repetitious to mention again here.

Reply:
ok